docs/troubleshooting-oauth2.md

```diff
@@ -41,25 +41,32 @@ OAuth resource [rabbitmq] not available. OpenId Discovery endpoint https://<the_
 #### Troubleshoot the issue
 
 These are the most common reasons for this issue:
-- The endpoint is either down or is not reachable (e.g. there is a firewall which blocks access).
-- The endpoint has a SSL certificate not trusted by the browser. 
-- The browser is blocking access due to [CORS](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) policy.
 
-The quickest way to identity the root cause is by opening the browser's javascript console and searching for `net::ERR_`. These are the two possible errors you may find:
-- `net::ERR_CONNECTION_REFUSED` The endpoint is down or is not reachable. 
-- `net::ERR_CERT_AUTHORITY_INVALID` The endpoint's SSL Certificate is not trusted by your browser. If you want to trust this certificate, click on the URL in the error message. The browser will prompt you to trust it. 
+ * The endpoint is either down or is not reachable (e.g. there is a firewall which blocks access)
+ * The endpoint has a TLS certificate not [trusted](./ssl#peer-verification) by the browser
```

Comment:
Use full word for e.g, i.e. in documentation, change it to "for example, there....", I don't you need to put the example in brackets...."not reachable, for example...."

Reply:
Add full stops at the end of the sentences

Reply:
The most important and valuable content of these two bulletpoints are the actual error code you have removed from your suggestion :) . I will keep the sentences as they are.